vendor/bower.json

```diff
@@ -1,12 +0,0 @@
-{
-  "name": "vendor",
-  "dependencies": {
-    "jquery-ujs": "~1.0.3",
-    "jquery-bootpag": "~1.0.7",
-    "lodash": "~3.10.0",
-    "d3": "~3.5.5",
```

Comment:
Please take a look of what is loaded via the assets server (at https://github.com/datacite/volpino/blob/master/app/views/layouts/application.html.erb#L17):

https://github.com/datacite/segugio/blob/master/source/javascripts/default.js

Reply:
We again might be able to load all assets from assets.datacite.org, maybe we need to add a file there.

Reply:
I've removed some of the dependencies that come in from assets server, I think this probably merits some more discussion at some point, but this is suitable for now.

Reply:
Sounds good. Volpino is our only Rails app with frontend code, and that is probably replaces by a Javascript frontend next year.